### PR TITLE
Dac6 1793: complete CheckYourAnswers

### PR DESCRIPTION
**Pull request by ers2015112**

### CODE REVIEW

File: test/controllers/CheckYourAnswersControllerSpec.scala [hunk 1]
```diff
@@ -20,7 +20,6 @@ import base.SpecBase
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
 import viewmodels.govuk.SummaryListFluency
-import views.html.CheckYourAnswersView
 
 class CheckYourAnswersControllerSpec extends SpecBase with SummaryListFluency {
```
Comment (soljeggers) [inline test/controllers/CheckYourAnswersControllerSpec.scala:24]: tests?

File: app/viewmodels/checkAnswers/DoYouHaveUTRSummary.scala [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object DoYouHaveUTRSummary  {
+
+  def row(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
+    answers.get(DoYouHaveUTRPage).map {
+      answer =>
+
+        val value = if (answer) "site.yes" else "site.no"
```
Comment (soljeggers) [inline app/viewmodels/checkAnswers/DoYouHaveUTRSummary.scala:33]: Could this be made reusable for all yesNo pages?

File: app/viewmodels/checkAnswers/BusinessWithoutIdAddressSummary.scala [hunk 3]
```diff
@@ -26,30 +26,32 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
...
+          } concat {
+            formatLine(answer.addressLine3)
+          } concat {
+            answer.addressLine4.fold("")(formatLine)
+          } concat {
+            answer.postCode.fold("")(formatLine)
+          } concat {
+            s"""<div class=govuk-margin-bottom-0>${HtmlFormat.escape(answer.country.description)}</div>"""
+            }
+
```
Comment (soljeggers) [inline app/viewmodels/checkAnswers/BusinessWithoutIdAddressSummary.scala:50]: Could this formatting or something similar be reused in the YourBusinessSummary object?

Comment (ers2015112) [reply app/viewmodels/checkAnswers/BusinessWithoutIdAddressSummary.scala:50]: The prototype for YourBusinessSummary doesn't use div's like this one it paragraphs, I don't know why

**Review by soljeggers — APPROVED** [review]

**Pull request merged**